lib/db/structs.proto

```diff
@@ -28,6 +28,7 @@ message FileInfoTruncated {
     uint32                permissions    = 4;
     int64                 modified_s     = 5;
     int32                 modified_ns    = 11;
+    uint64                modified_by    = 12;
```

Comment:
Nullable false is the default anyway for uint64 and is compatible, it's just that witht he custom type the Go protobuf compiler would want to make it a *ShortID otherwise which we don't want. So yeah, nullable=false.